### ISSUE TITLE
kicks me out 

### ISSUE THREAD

**Issue by georgianmapper2** [Crunnie/Fortnite-iOS-Archive#19]
in chapter 2 season2/3 it kicks me out of the bus every time I jump

**Comment by georgianmapper2** [comment]
another thing is that I can’t even download anything because every time I try and download it my screen goes black

**Comment by georgianmapper2** [comment]
also season 13 doesn’t work at all

**Comment by Statedmite** [comment]
Yeah it does not work season 3 but even season 1 does not work and season two is opening but it never loads same with chapter one
